assets/src/edit-story/app/media/utils/getResourceFromLocalFile.js

```diff
@@ -120,21 +129,48 @@ const getVideoResource = async (file) => {
   });
 };
 
+const createPlaceholderResource = (properties) => {
+  return createLocalResource({ ...properties, isPlaceholder: true });
+};
+
+const getPlaceholderResource = (file) => {
+  const type = getTypeFromMime(file.type);
+  const mimeType = type === 'image' ? 'image/png' : 'video/mp4';
+
+  // The media library requires resources with valid mimeType and dimensions.
+  return createPlaceholderResource({
+    type: type || 'image',
+    mimeType: mimeType,
+    src: '',
+    width: PAGE_WIDTH,
+    height: PAGE_HEIGHT,
```

Comment:
```suggestion
    height: PAGE_HEIGHT,
    alt: fileName,
    title: fileName,
```

Why not some default alt and titles as well.

Reply:
I've kept it pretty minimal because these values will get overridden during upload anyway.

Reply:
For the most part I agree, but an alt is useful for accessibility. It stop an image tag for updating it's alt value.